src/tiffimage.cpp

```diff
@@ -188,7 +188,7 @@ namespace Exiv2 {
         // recursively print the structure to /dev/null to ensure all metadata is in memory
         // must be recursive to handle NEFs which stores the raw image in a subIFDs
         std::ofstream devnull;
-        printStructure(devnull,kpsRecursive,0);
+        printStructure(devnull,kpsRecursive);
```

Comment:
I'd rather keep it for consistency with the other `*image.cpp` files where depth is explicitly specified.